### PR TITLE
Add dashboard creation section for Beacons

### PR DESCRIPTION
**Pull request by bschoeneweis**
## What?
Adding the dashboard as a way to create beacons.

## Screenshot
image

### CODE REVIEW

File: docs/beacons.mdx [hunk 1]
```diff
@@ -56,14 +56,18 @@ Beacon entry and exit events are sent to the SDK, to [webhooks](/integrations#we
...
 
 Like [geofences](/geofences), you also specify the metadata for beacons when you create them, including *tag* (a group for the beacon, e.g., `store-register`), external ID (an external ID for the beacon that maps to your internal database, e.g., `123-1`), *description* (a display name for the beacon, e.g., `Store #123 - Register #1`).
 
 You also specify the identifiers used to range the beacon, including `uuid`, `major`, and `minor`, as well as an approximate latitude and longitude used to sync nearby beacons (within 1 kilometer) to the SDK.
 
 **Beacons should be uniquely referenced by tag and external ID, assigned by you when a beacon is created. To disable or rotate identifiers for a beacon, re-import the beacon with the same tag and external ID.**
 
+### Dashboard
+
+To create a beacon via the dashboard, go to the [Beacons page](https://radar.com/dashboard/beacons) and click the _New_ button. Enter a description, type, tag, external ID, UUID, major, minor, optional metadata, and a latitude and longitude. Click _Create_ to create the beacon.
```
Comment (jsani-radar) [inline docs/beacons.mdx:69]: 1st / 3rd sentences are good - can we have the 2nd be consistent with the [geofences dashboard creation section](https://radar.com/documentation/geofences#dashboard)?

"Optionally search for an address or place, then enter a type, tag, external ID, UUID, major, minor, and optional metadata."

**Review by jsani-radar — COMMENTED** [review]
small nit - looking good otherwise, we'll merge this when the feature rolls out

File: docs/beacons.mdx (hunk 1 above)
Comment (bschoeneweis) [reply docs/beacons.mdx:69]: Oh yeah good catch, forgot to update after the wave of changes.

**Review by jsani-radar — APPROVED** [review]
🙏🏽

**Pull request merged**